### ISSUE TITLE
Possible compromise of the extension

### ISSUE THREAD

**Issue by abishekmuthian** [ken107/read-aloud#232]
I saw a hyperlink with the text 'India success [story]' on the footer of the popup of the extension. Clicking the URL leaded to the YouTube video titled '[Home ivermectin based kits in India](https://www.youtube.com/watch?v=eO9cjy3Rydc)'

The hyperlink is displayed through the announcement routine here- 

https://github.com/ken107/read-aloud/blob/8770c74f323392ba113f50eb1c4014d07b28ffd9/js/popup.js#L213

Data fetched from - https://support.readaloud.app/read-aloud/announcement matches what I saw,

```
{"id":"ivermectin","text":"India success [story]","link":"https://youtu.be/eO9cjy3Rydc","period":1440}
```

Is the servers compromised? Or is this an intentional promotion of Ivermectin by the developer/publisher?

**Comment by WolfIcefang** [comment]
I am also seeing this. It seems to only appear on the first launch of the extension, then it reverts to the traditional "report issues" button.
![image](https://user-images.githubusercontent.com/38160450/134535852-b3564dbf-ac41-4cf8-8439-c6977d6b3e6d.png)
removing and reinstalling the extension will make the text appear.
I just installed version 1.46.0 today onto Firefox 92.0 (64-bit) (and on Windows 10, if that matters.); I've never used this TTS software before.

**Comment by ken107** [comment]
We make this announcement not to promote Ivermectin, but to bring awareness to a story that has so far been suppressed.  The unequivocal success of India in squashing the Delta wave despite having only 14% full vaccination rate is unusual, when other countries are struggling to contain it even with most of their population vaccinated.

**Comment by abishekmuthian** [comment]
> We make this announcement not to promote Ivermectin, but to bring awareness to a story that has so far been suppressed. 

I really hoped that your server was as compromised and  this was not a deliberate abuse of trust.

I really didn't want to talk about Ivermicitin as such and my focus was on the security aspect of the extension but I feel it would be dishonest on my part if I don't refute your factually incorrect statements since this concerns with people's life.

> The unequivocal success of India in squashing the Delta wave despite having only 14% full vaccination rate is unusual, when other countries are struggling to contain it even with most of their population vaccinated.

That's an absolute misinformation. I'm an Indian, I recovered from COVID, I lost people to COVID and I've been following every aspect of the pandemic.

- First and foremost [Indian Government has removed Ivermicitin from the list of medications for treatment since june](https://www.livemint.com/news/india/govt-removes-ivermectin-other-popular-drugs-from-covid-19-treatment-list-check-revised-guidelines-11623074154396.html) and if there was any evidence to prove that it had any positive effect on the treatment of COVID-19 it wouldn't have done so.
- Very few states included everything from homeopathy, ayurveda, siddha to Ivermectin in their at home care package but quickly shifted to evidence based medication. One such state  is being talked about in the video you're linking to in the extension has a population of just 1.59 Million (India's overall population is 1.3+ billion).
- Finally the drop in cases in India is due to various reasons incl. large number of population already being infected, severe control measures via lockdowns, awareness and now the vaccination drive.

Please don't use India to propel your agenda and especially doing it via a browser extension trusted (was) by thousands of users including those with accessibility needs like me. Besides I'm absolutely sure that this is a violation of Chrome web store and Firefox add-ons terms.

**Comment by ken107** [comment]
You claimed that these are the reasons India is so successful in managing the pandemic:

- Large number of population already infected
https://www.worldometers.info/coronavirus/#countries
If you sort by "Total cases/Million population" India is number 126.  In other words, India has one of the lowest percentage of population already infeccted with Covid.

- Severe control measures via lockdown 
It's false to assume that other countries aren't locking down as severely as India.  Vietnam struggles to contain the Delta wave despite extreme measures.

- Awareness and now the vaccination drive
Are you saying that Indian people are more aware that there is a pandemic than other countries?  In terms of vaccination, India quashed the Delta wave back in early June, when their full-vaccinated rate is 3% (https://archive.is/tx6nQ)

In summary, your claims are completely without evidence or support.  On the other hand, you're helping to suppress actual evidence and data.  You clearly did not watch the video by Dr. Campbell, who treated the subject impartially and never once recommended Ivermectin.  He merely reports the data as he gathers it, and raise legitimate questions that need answers.  I'm merely helping to bring awareness to the issue because people would not get it from more mainstream channels.

**Comment by abishekmuthian** [comment]
> In other words, India has one of the lowest percentage of population already infeccted with Covid.

My statement was based upon on [ICMR sero survey: One in five Indians exposed to Covid-19](https://www.bbc.com/news/world-asia-india-55945382) on which the Govt. policies are made.

> It's false to assume that other countries aren't locking down as severely as India.

The context of your statement on the extension is 'India success [story]'. What I think about lock downs in other countries is irrelevant. 

> Are you saying that Indian people are more aware that there is a pandemic than other countries?

Again what I think about awareness of people in other countries is irrelevant.

Coming back to the extension itself,

**If you wanted to promote your anti-vaccination agenda through your browser extension put it up as the purpose of the extension everywhere it's available.** Many visually impaired people use this extension, You are putting up a deceptive statement dynamically for them to click when using the TTS feature on a web page. What is stopping you from inserting your statement within the text of the page itself?

**Comment by ken107** [comment]
It is the very same ICMR that recommended Ivermectin in its April 20 guideline for covid management, 3 weeks before the daily case count takes a dive.  In the video, the Dr. makes clear that this is only a correlation, does not prove causation.  He did not try to prove that it was due to Ivermectin, but you did not prove that it was due to something else.

22% seroprevalence is not a high number.  Admittedly this means that the official case count, as reported by virtually all news outlets, are 10x lower than the actual number.  But nevertheless, something protected the remaining 1 billion people, that resulted in the precipitous decline beginning on May 7.  Unless there is definitive proof, we will continue to ask questions.  Not promoting Ivermectin, but asking legitimate questions.

**Comment by abishekmuthian** [comment]
So you're doubling down on your agenda without addressing the deceptive practice. This leaves with no choice but to report the extension on [Chrome Webstore](https://chrome.google.com/webstore/detail/read-aloud-a-text-to-spee/hdhinadidafjejdhmfkjgnolgimiaplp) and [Firefox add-ons](https://addons.mozilla.org/en-US/firefox/addon/read-aloud/) for the deceptive practice.

**Comment by ken107** [comment]
Look, I took off the announcement.  If indeed the actual number of cases and deaths is **10 times** the official reported death count, unimaginable how incompetent your government is.  Then my statements, which are based on the data I can gather, are wrong, and India's not a success.

**Comment by aviwad** [comment]
I am from India and looking at the impact of Covid in the past year, it’s clear to say you are spreading dangerous misinformation considering that the horse medication ivermectin is not recommended for Covid at all. India slowly recovered due to various other circumstances. I will be reporting your extension

try gaining some brain cells next time before embedding your cancerous anti-vax propaganda on your extension.

**Comment by abishekmuthian** [comment]
> Look, I took off the announcement. If indeed the actual number of cases and deaths is **10 times** the official reported death count, unimaginable how incompetent your government is. Then my statements, which are based on the data I can gather, are wrong, and India's not a success.
> 
> I understand how you can undercount the number of cases due to lack of testing. But how do you undercount the number of deaths? By 10 times?

The contention was never whether 'India is a success story or not', The contention was you using deceptive practice to promote your agenda by using 'India'.

If you have really rolled-back your deceptive practice, Then I appreciate it. I'm not confident to try your extension again to verify it.
